mllib/src/main/scala/org/apache/spark/mllib/tree/loss/AbsoluteError.scala

```diff
@@ -61,4 +61,18 @@ object AbsoluteError extends Loss {
       math.abs(err)
     }.mean()
   }
+
+  /**
```

Comment:
No need for doc; it will be inherited from the overridden method  (here and in other 2 loss classes)